go/appencryption/README.md

```diff
@@ -14,6 +15,11 @@ Application level envelope encryption SDK for Golang with support for cloud-agno
   * [Documentation](#documentation)
   * [Development Notes](#development-notes)
 
+### Installation
+```bash
+go get github.com/godaddy/asherah/go/appencryption
+```
+
```

Comment:
This isn't needed as the introduction of go modules eliminated the need for explicit `go get` installs. The `go` command will automatically install dependencies (and update go.mod) whenever a new package import is found. That said, it probably makes sense to update the Quick Start section to include the appencryption package imports.